app/src/main/res/values-en/strings.xml

```diff
@@ -1377,4 +1377,106 @@
+    <string name="event_type_elearning">online lesson</string>
+    <string name="menu_messages_config">Messages settings</string>
+    <string name="message_delete">Delete</string>
+    <string name="message_reply">Reply</string>
+    <string name="message_forward">Forward</string>
+    <string name="discard">Discard</string>
+    <string name="messages_compose_draft_saved">Saved draft</string>
+    <string name="messages_tab_draft">Drafts</string>
+    <string name="messages_compose_send_long">Send a message</string>
+    <string name="notes_button">Notes</string>
+    <string name="notes_list_dialog_title">Notes</string>
+    <string name="notes_added_by_you_format">Added %1$s</string>
+    <string name="notes_type_attendance">Attendance</string>
+    <string name="notes_type_behavior">Behavior</string>
+    <string name="notes_type_day">Day</string>
+    <string name="notes_type_event">Event</string>
+    <string name="notes_type_lesson">Lesson</string>
+    <string name="notes_type_message">Message</string>
+    <string name="notes_details_id">Note ID</string>
+    <string name="notes_details_owner_id">Owner ID</string>
+    <string name="notes_type_announcement">Announcement</string>
+    <string name="styled_text_dialog_title">Edit text</string>
+    <string name="login_qr_decoding_error">The QR code doesn\'t seem correct</string>
+    <string name="notes_no_data_hint">You can add notes and share them with the class using the Add button.</string>
+    <string name="legend_notes_added">{cmd-playlist-edit} notes added</string>
+    <string name="legend_notes_added_replaced">{cmd-playlist-edit} notes added {cmd-swap-horizontal} has been replaced with a note</string>
+    <string name="notes_no_data">No notes were added</string>
+    <string name="messages_compose_discard_draft_text">Do you want to discard the saved version of the message? This will also cancel your changes and delete your message.</string>
+    <string name="messages_compose_discard_draft">Discard draft</string>
+    <string name="messages_compose_draft_discarded">Discarded draft</string>
+    <string name="messages_compose_discard_draft_title">Discard draft</string>
+    <string name="notes_details_dialog_title">Note</string>
+    <string name="notes_editor_topic">Note title (optional)</string>
+    <string name="notes_editor_body">Note body</string>
+    <string name="notes_editor_share_hint">The classmates will see your note next to this item.</string>
+    <string name="notes_editor_replace_hint">The content of the note will be visible instead of the original content of the item.</string>
+    <string name="notes_editor_body_error">Enter the note text</string>
+    <string name="color_none">None</string>
+    <string name="notification_type_new_shared_note">Note shared</string>
+    <string name="notes_editor_confirmation_text">Do you want to delete this note? It will be deleted on your device and on other people in the class.</string>
```

Comment:
```suggestion
    <string name="notes_editor_confirmation_text">Do you want to delete this note? It will be deleted on your device and on classmates' devices.</string>
```